mlir/test/Examples/nvgpu/tools/nvdsl.py

```diff
@@ -0,0 +1,462 @@
+from enum import Enum
+import functools, sys, ctypes, os, errno
+import numpy as np
+from functools import partialmethod
+from mlir import ir
+from mlir.dialects import arith, func, gpu, memref, nvgpu, scf, nvvm
+from mlir.extras import types as T
+from mlir import runtime as rt
+from tools import nvgpucompiler
+
+MLIR_DYNAMIC = -9223372036854775808
```

Comment:
Could we avoid this? This should be an internal detail

Reply:
How can I avoid that? This is the value that mlir uses to select static values